docs/next-release-notes.md

```diff
@@ -239,6 +362,10 @@ https://github.com/apache/druid/pull/12177
 
 ## Operations
 
+### Deploying Druid
+
+There is now a Python installation script available for Druid that simplifies deployments that don't fit the parameters of the singel-server profiles, such as the nano or micro-quickstart profiles. This `start-druid` script lets you override Druid settings to fit your needs. For more information, see [Single server deployment](https://druid.apache.org/docs/latest/operations/single-server.html).
```

Comment:
The new script simplifies deployments for all cases, not just the ones that don't fit the existing single-server profiles. We could probably rephrase this section as follows:

```suggestion
The new `start-druid` script greatly simplifies starting any combination of Druid services on a single-server. It comes pre-packaged with the required configs and can be used to launch a fully functional Druid cluster simply by invoking `./start-druid`. For the experienced Druids, it also gives you complete control over the runtime properties and JVM arguments to have a cluster that exactly fits your needs.

The `start-druid` script deprecates the existing profiles such as `start-micro-quickstart` and `start-nano-quickstart`. These profiles may be removed in future releases. For more information, see [Single server deployment](https://druid.apache.org/docs/latest/operations/single-server.html).
```

@vogievetsky , does this capture our sentiments adequately?

Reply:
I [moved this section to the highlights](https://github.com/apache/druid/pull/13429#discussion_r1050809228) and also used the text from this comment.